__tests__/index.test.js

```diff
@@ -141,6 +141,30 @@ describe('GET /api/articles', () => {
   })
 })
 
+describe('GET /api/articles (queries)', () => {
+  it('should accept a topic query and returns articles only with this topic', () => {
+    return request(app)
+    .get('/api/articles?topic=cats')
+    .expect(200)
+    .then((res) => {
+      const articles = res.body.articles
+      expect(articles.length).toBe(1)
+      articles.forEach((article) => {
+        expect(article.topic).toBe('cats')
+      })
+    })
+  })
+  it('should return nothing if given a non-existent topic', () => {
```

Comment:
If this is a non-existent topic, we would expect a 404 and appropriate error message, not a 200